buildSrc/buildSrc/src/main/kotlin/Deps.kt

```diff
@@ -1,7 +1,7 @@
 object Deps {
 
-    private const val kotlinVersion = "1.3.50"
-    private const val coroutinesVersion = "1.3.2"
+    private const val kotlinVersion = "1.3.60"
+    private const val coroutinesVersion = "1.3.2-native-mt-1"
```

Comment:
Version looks strange. Also they have `1.3.2-1.3.60` version, maybe better to use this one?
https://bintray.com/kotlin/kotlinx/kotlinx.coroutines

Reply:
Yeah, this is the new one